src/bitbucket/BitbucketMerger.ts

```diff
@@ -11,12 +11,12 @@ export class BitbucketMerger {
 
   constructor(private baseUrl: string) {}
 
-  attemptMerge = async (prId: number, message: string) => {
+  attemptMerge = async (prId: number, message: string, mergeStrategy?: IMergeStrategy) => {
     const endpoint = `${this.baseUrl}/pullrequests/${prId}/merge`;
     const body = {
       close_source_branch: true,
       message,
-      merge_strategy: 'merge_commit',
+      merge_strategy: mergeStrategy?.split('-').join('_') || 'merge_commit',
```

Comment:
can you provide some insight into what the difference is between `merge_commit` and `merge-commit`? I'm seeing both used across this PR

Reply:
Bitbucket API uses snake case (merge_commit) and I saw that in landkid we follow hyphen case (merge-commit). This converts the casing.